docs/pages/serving_a_federation.mdx

```diff
@@ -0,0 +1,209 @@
+A Pelican federation requires both *director* and *reigstry* to be set up first before any origin or cache server can join
+the federation or any client to access objects in the federaiton.
+
+## Install Pelican
+
+Grab the appropriate binary for your system from the [pelican repository](https://github.com/PelicanPlatform/pelican/releases). For more details, see [the installation instructions](/install.mdx).
+
+## Prepare TLS Credentials
+
+Pelican servers use `https` for serving its web UI and handling internal http requests. `https` requires a set of credential files in place to work, including:
+
+- A valid TLS certificate
+- The private key associated with the certificate
+- The Intermediate Certificate or the chain file, that establishes the trust chain to a root certificate
+
+> For local development and testing, you may skip setting up TLS credentials by setting configuration parameter `TLSSkipVerify` to `true`. You should **NOT** set this for production.
+
+You need to contact a Certifcate Authority (CA) who owns the root certificate for getting these credentials. One popular CA that provides free TLS certifactes is [Let's Encrypt](https://letsencrypt.org/). You may follow [their guide](https://letsencrypt.org/getting-started/) to obtain the credentials listed above. **Note that you need to have a valid domain before proceeding.**
+
+Once you go through the process, locate your credential files and set the following parametes to the file locations, or copy the credential files to the default location of the parameter:
+
+- `Server.TLSCertificate`
+  - The certicate file from Let's Encrypt, usually named as `example.com.crt` or `example.com.pem`
+  - Default location: `~/.config/pelican/certificates/tls.crt` for non-root users or `/etc/pelican/certificates/tls.crt` for root users.
+
+- `Server.TLSKey`
+  - The private key corresponding to the TLS certificate, usually named as `example.com.key`.
+  - Default location: `~/.config/pelican/certificates/tls.key` for non-root users or `/etc/pelican/certificates/tls.key` for root users.
+
+- `Server.TLSCACertificateFile`
+  - The intermediate certificate from Let's Encrypt to establishe the trust chain to a root certificate, usually named as `letsencrypt-intermediate.pem` or `chain.pem`
+  - Default location: `~/.config/pelican/certificates/tlsca.pem` for non-root users or `/etc/pelican/certificates/tlsca.pem` for root users.
+
+Since your TLS certificate is associated with your domain name, you will need to change the default hostname of Pelican server to be consistent. Set `Server.Hostname` to your domain name i.e. `example.com`.
+
+## Serve a Registry
+
+Pelican _registry_ manages namespace registration and public keys distribution. An origin or cache will automatically attempt to register their namespace at their start. A federation user may also register a namespace without running a storage server by visiting the registry website.
```

Comment:
Sometimes this is using * registry *  and sometimes it's using _ registry _ . While it might look the same, I think we should be internally consistent.